ci-operator/step-registry/openshift/microshift/infra/volume-create/openshift-microshift-infra-volume-create-commands.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -euo pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+IP_ADDRESS="$(cat ${SHARED_DIR}/public_address)"
+HOST_USER="$(cat ${SHARED_DIR}/ssh_user)"
+INSTANCE_PREFIX="${HOST_USER}@${IP_ADDRESS}"
+
+echo "Using Host $IP_ADDRESS"
+
+mkdir -p "${HOME}/.ssh"
+cat <<EOF >"${HOME}/.ssh/config"
+Host ${IP_ADDRESS}
+  User ${HOST_USER}
+  IdentityFile ${CLUSTER_PROFILE_DIR}/ssh-privatekey
+  StrictHostKeyChecking accept-new
+  ServerAliveInterval 30
+  ServerAliveCountMax 1200
+EOF
+chmod 0600 "${HOME}/.ssh/config"
+
+cat > "${HOME}"/volume-create.sh <<EOF
+#!/bin/bash
+set -xeuo pipefail
+
+sudo dnf install -y lvm2 jq
+pv_location=\$(sudo lsblk -Jd | jq -r '.blockdevices[] | select(.size == "200G") | "/dev/\(.name)"')
```

Comment:
In the course of working on the existing steps, I found the use of here-documents to generate script files introduced a lot of opportunities for errors when not escaping the right `$`. Can we put scripts like this into their own files here in this repo and then just copy them to the remote host to run them?

Reply:
The CI automation is looking for exactly one script file per step-ref, so there's not any good way to have a library of files per step. Everything has to be included in the `*-commands.sh` file alone.

In this sub-script's case, suppress variable expansion & substitution by quoting the delimiter (`"EOF"`).  There aren't any variables that need to be expanded in the current env anyway.